vio/vio/swagger/views/image/views.py

```diff
@@ -105,7 +105,9 @@ def post(self, request, vimid, tenantid):
                     return Response(data=rsp, status=status.HTTP_200_OK)
 
             param = image_utils.req_body_formatter(req_body)
-            image = image_instance.create_vim_image(vimid, tenantid, **param)
+            image = image_instance.create_vim_image(vimid, tenantid,
+                                                    imagePath=req_body.get('imagePath'),
```

Comment:
create_vim_image function does not get imagePath if you don't modify the OperateImage.py